pyiron_atomistics/atomistics/structure/factory.py

```diff
@@ -51,6 +53,64 @@
 
 
 class StructureFactory(PyironFactory):
+    class ase:
+        def cut(*args, **kwargs):
+            """
+            Returns an ASE's cut result, wrapped as a `pyiron_atomistics.atomstic.structure.atoms.Atoms` object.
+
+            ase.build.cut docstring:
+
+            """
+            s.publication_add(publication_ase())
+            return ase_to_pyiron(ase_cut(*args, **kwargs))
```

Comment:
In principle, this should work even if we pass a pyiron `Atoms` object to the function since it is now derived from `ase`. So there is no need for `ase_to_pyiron` (which can be slow, see: https://github.com/pyiron/pyiron_atomistics/issues/48). 

```suggestion
            return ase_cut(*args, **kwargs)
```

Reply:
Yes, the ase `cut` and `stack` methods work with our structure class.